lua/luasnip/util/path.lua

```diff
@@ -51,18 +51,15 @@ function Path.read_file(path)
 	return buf
 end
 
-local MYCONFIG_ROOT = vim.env.MYVIMRC
--- if MYVIMRC is not set then it means nvim was called with -u
--- therefore the first script is the configuration
--- in case of calling -u NONE the plugin won't be loaded so we don't
--- have to handle that
+local MYCONFIG_ROOT = vim.env.MYCONFIG_ROOT
 
 if not MYCONFIG_ROOT then
-	MYCONFIG_ROOT = vim.fn.execute("scriptnames"):match("1: ([^\n]+)")
+	if vim.env.MYVIMRC then
+		MYCONFIG_ROOT = vim.env.MYVIMRC:gsub(("%s[^%s]+$"):format(sep, sep), "")
```

Comment:
What do you think of using `fnamemodify` here?
Would make this a bit more readable